cohesive_energy.py

```diff
@@ -0,0 +1,28 @@
+from ase.md.velocitydistribution import (MaxwellBoltzmannDistribution,Stationary,ZeroRotation)
+from ase.md.verlet import VelocityVerlet
+from asap3 import Trajectory
+from ase import units
+from equilibriumCondition import equilibiriumCheck
+
+
+
+def cohesive_energy(atoms,iterations):
+    """The cohesive_energy function takes argument atoms, that is the simulated material,
+    and makes another simulation for the material at temperature 0 K.
+    It the looks extracts the potential energy which is the cohesive energy for the material"""
+    MaxwellBoltzmannDistribution(atoms, temperature_K=0)
+    Stationary(atoms)
+    ZeroRotation(atoms)
+    coh_dyn = VelocityVerlet(atoms, 5 * units.fs)
+    coh_traj = Trajectory(
+                "coh_E.traj", 
+                "w", 
+                atoms, 
+                properties="energy, forces"
+            )
```

Comment:
This should include another argument master=true to be able to be run in parallell.

Reply:
Fixed!